integration-sqs-to-lambda-sample-code/Function.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Amazon.Lambda.Core;
+using Amazon.Lambda.SQSEvents;
+
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace SqsIntegrationSampleCode
+{
+    public class Function
+    {
+        /// <summary>
+        /// Default constructor. This constructor is used by Lambda to construct the instance. When invoked in a Lambda environment
+        /// the AWS credentials will come from the IAM role associated with the function and the AWS region will be set to the
+        /// region the Lambda function is executed in.
+        /// </summary>
+        public Function()
+        {
+
+        }
+
+        /// <summary>
+        /// This method is called for every Lambda invocation. This method takes in an SQS event object and can be used 
+        /// to respond to SQS messages.
+        /// </summary>
+        /// <param name="evnt"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public async Task<string> FunctionHandler(SQSEvent evnt, ILambdaContext context)
+        {
+            foreach (var message in evnt.Records)
```

Comment:
Can we demonstrate how to handle failures, or at least put comments in as to how to consider failure handling

Reply:
this snippet is for sample codes under aws docs, the other version that handles batchItemFailure is demonstrated under that you reviewed earlier this week https://github.com/aws-samples/serverless-snippets/pull/42